package.json

```diff
@@ -7,7 +7,8 @@
     "test": "jest --verbose",
     "start": "webpack-dev-server",
     "build": "webpack",
-    "check-types": "tsc --noEmit"
+    "check-types": "tsc --noEmit",
+    "lint": "find js -name '*.ts' | grep -v '\\.d\\.ts' | xargs tslint"
```

Comment:
You could write this as one command `find js -name '*.ts' ! -name '*.d.ts' -exec tslint {} +` and avoid the horror that is dealing with filenames passed between processes